docs(migration): update to use new `runtimeConfig` (#4642)
nuxt/framework: docs/content/migration/8.runtime-config.md

## Migration

> 1. Add any environment variables you use in your app to `runtimeConfig` in `nuxt.config.ts`.

Reviewer: ```suggestion
1. Add any environment variables that you use in your app to the `runtimeConfig` property of the `nuxt.config` file.
```

export default defineNuxtConfig({
  runtimeConfig: {

>     secretKey: '', // variable that can only be accessed on server-side

Reviewer: ```suggestion
    secretKey: '', // variable that can only be accessed on the server side
```

    public: {

>       BASE_URL: process.env.BASE_URL || 'https://nuxtjs.org' // variable that can also be accessed on the client-side

Reviewer: ```suggestion
      BASE_URL: process.env.BASE_URL || 'https://nuxtjs.org' // variable that can also be accessed on the client side
```
